spec/features/user_visit_genre_page_spec.rb

```diff
@@ -0,0 +1,15 @@
+require 'rails_helper'
+
+describe "User views genre page" do
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.